go/tasks/plugins/k8s/kfoperators/common/config.go

```diff
@@ -0,0 +1,32 @@
+package common
+
+import (
+	"time"
+
+	pluginsConfig "github.com/flyteorg/flyteplugins/go/tasks/config"
+	"github.com/flyteorg/flytestdlib/config"
+)
+
+//go:generate pflags Config --default-var=defaultConfig
+
+var (
+	defaultConfig = Config{
+		Timeout: config.Duration{Duration: 1 * time.Minute},
+	}
+
+	configSection = pluginsConfig.MustRegisterSubSection("common", &defaultConfig)
```

Comment:
Does this result in configuration like:
```
plugins:
    common:
         timeout: 10s
     k8s:
         ...
     ....
```
IMO `common` is not descriptive enough here. Maybe something like `kf-operator`. Thoughts?

Reply:
That makes sense, updated it, thanks.